### ISSUE TITLE
conformer MLS-pt

### ISSUE THREAD

**Issue by flaviofafe1414** [TensorSpeech/TensorFlowASR#193]
Hi, I'm trying to train the conformer model with the MLS Pt dataset, but I found some strange metrics in their order of magnitude.

greedy_wer: 0.9748954176902771
greedy_cer: 0.9024566411972046
beamsearch_wer: 1.0
beamsearch_cer: 1.0

Due to its loss curve, I believe that the model is not learning, but when I put just one example for the test, it predicts correctly. 
![Captura de tela de 2021-04-30 15-14-50](https://user-images.githubusercontent.com/43591522/116737374-8a5f0080-a9c7-11eb-8f66-3f54e9280d90.jpg)

GROUNDTRUTH
foice cortou o pescoço do animal a cabeça ficou segura na carne da vítima e das artérias rotas jorrava o sangue não havia mais cães a matar o terreiro ficara alastrado de corpos decepados mutilados de membros esparsos os homens maltratados doloridos deitavam

TRANSCRIPT
foice cortou o pescoço do animal a cabeça ficou segura na carne da vítima e das artérias rotas jorrava o sangue não havia mais cães a matar o terreiro ficara alastrado de corpos decepados mutilados de membros esparsos os homens maltratados doloridos deitavam

When I place all test dataset for evaluation, the phrases are correct, but a GROUNDTRUTH phrase corresponds to a TRANSCRIPT in another position

**Comment by usimarit** [comment]
@flaviofafe1414 the test result file is a tsv with headers: PATH DURATION GROUNDTRUTH GREEDY BEAMSEARCH
So your result will be in GREEDY and BEAMSEARCH, the GROUNDTRUTH is the label to be compared to calculate metrics, so the GROUNDTRUTH in the result and TRANSCRIPT in the dataset are the same, so you may be mistaken 😄

**Comment by flaviofafe1414** [comment]
oh ok, I found my mistake. 

thanks
